homeassistant/components/esphome/light.py

```diff
@@ -212,8 +222,14 @@ async def async_turn_on(self, **kwargs: Any) -> None:
         if (transition := kwargs.get(ATTR_TRANSITION)) is not None:
             data["transition_length"] = transition
 
```

Comment:
You'll need to fix the `min_ct` and `max_ct` above as well

```
-                    min_ct = self.min_mireds
-                    max_ct = self.max_mireds
+                    min_ct = self.min_color_temp_kelvin
+                    max_ct = self.max_color_temp_kelvin
```

Reply:
This would need to be converted

Reply:
Might as well leave it since its in the static info anyways and thats not going to change.